src/server/transaction.h

```diff
@@ -124,6 +126,10 @@ class Transaction {
   void LogJournalOnShard(EngineShard* shard, journal::Entry::Payload&& payload,
                          uint32_t shard_cnt) const;
 
+  // Rewrite parts of underlying command.
+  // Can be called only once per transaction.
```

Comment:
what about multi command transaction

Reply:
Indeed, it the rewrite stash needs to be cleared there